mica/archive/ocat_target_table.py

```diff
@@ -0,0 +1,28 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import numpy as np
+from astropy.table import Table
+from pathlib import Path
+from mica.common import MICA_ARCHIVE
+
+OCAT_TARGET_TABLE = Path(MICA_ARCHIVE) / 'ocat_target_table.h5'
+
+def get_ocat_target_table(datafile=None):
+    """
+    Read the ocat target table h5 data product.
+
+    :param datafile: h5 ocat target table data file.
+                     Defaults to MICA_ARCHIVE/ocat_target_table.h5
+    :returns: astropy table of target table
+    """
+    if datafile is None:
+        datafile = OCAT_TARGET_TABLE
+
+    dat = Table.read(datafile)
+
+    # Decode bytes to strings manually.  Perhaps fixed in newer versions of numpy.
```

Comment:
Definitely fixed in newer versions of numpy.

Reply:
Hah.  The "perhaps" was more meant to suggest that I have no idea what we'll use in ska3, but fair.